.github/workflows/publish.yml

```diff
@@ -0,0 +1,18 @@
+name: publish
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "12.x"
```

Comment:
Should this be `14.x` to match the latest LTS and the version we are on?

Reply:
Technically, it doesn't really matter because we run our PR checks across all versions 12.x of `node.js` and as far as I know the node version we use for publishing don't need to match the version of node we are using.

Now that I'm thinking about it, I guess it doesn't hurt to use a newer version that has a longer LTS duration.